cairo/ethereum/cancun/trie.cairo

```diff
@@ -332,10 +347,47 @@ func encode_node{range_check_ptr, bitwise_ptr: BitwiseBuiltin*, keccak_ptr: Kecc
 //         // trie._data[key] = value
 // }
 
-// func trie_get(trie: Trie[K, V], key: K) -> V {
-//     // Implementation:
-//     // return trie._data.get(key, trie.default)
-// }
+func trie_get_TrieAddressAccount{poseidon_ptr: PoseidonBuiltin*, trie: TrieAddressAccount}(
```

Comment:
why `trie_get_TrieAddressAccount` and not `trie_get_account` or `trie_get_account_from_address`

Reply:
tbh it's fine like this

Reply:
consistency with type name as we dont have generics